.storybook/main.js

```diff
@@ -1,8 +1,50 @@
+const path = require('path');
+const dotenv = require('dotenv');
+
+// ----------------------------------------
+// Storybook custom env var parsing
+
+const previewDotenv = dotenv.config({
+	path: path.resolve(__dirname, '../.env.preview.info'),
```

Comment:
Currently the only env vars being complained about are in `.env.preview.info`, so just gonna go with that for now.

Reply:
Sounds good.